src/middlewared/middlewared/alert/source/iscsi.py

```diff
@@ -51,3 +51,57 @@ async def check(self):
 
         if ips:
             return Alert(ISCSIPortalIPAlertClass, ', '.join(ips))
+
+
+class ISCSIAuthSecretInvalidCharAlertClass(AlertClass):
+    category = AlertCategory.SHARING
+    level = AlertLevel.WARNING
+    title = 'iSCSI Authorized Access has an invalid character'
+    text = 'The iSCSI Authorized Access with Group ID %(tag)d and %(userfield)s %(user)r has a %(field)s containing the invalid character: %(char)r.'
+
+
+class ISCSIAuthSecretWhitespaceAlertClass(AlertClass):
+    category = AlertCategory.SHARING
+    level = AlertLevel.WARNING
+    title = 'iSCSI Authorized Access has leading or trailing whitespace'
+    text = 'The iSCSI Authorized Access with Group ID %(tag)d and %(userfield)s %(user)r has a %(field)s containing leading or trailing whitespace.'
+
+
+class ISCSIAuthSecretAlertSource(AlertSource):
+    schedule = IntervalSchedule(timedelta(hours=24))
+    run_on_backup_node = False
+    INVALID_CHARS = '#'
```

Comment:
Why define this again here in the class instead of just importing `INVALID_CHARACTERS` from `iscsi_.auth.py`?